app/code/Magento/Catalog/Model/Product/Option/Type/Date.php

```diff
@@ -355,7 +356,8 @@ public function getYearStart()
      */
     public function getYearEnd()
     {
-        $_range = explode(',', $this->getConfigData('year_range'));
+        $_range = $this->getConfigData('year_range')
```

Comment:
please use same approach\code style in both cases

Reply:
I made changes, please, check.
Thanks!